src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -215,6 +215,34 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
             tableBodyHtml.append(tableRow + Const.EOL);
         }
         
+        StringBuilder mobileHtml = new StringBuilder();
+
+        String mobilePanelTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL;
+        String mobilePanelFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL_FRAGMENT;
+
+        for(int j = 0 ; j < numOfRubricSubQuestions ; j++) {
```

Comment:
hey, we should use <code>j</code> as the inner loop variable, <code>i</code> should be the outer loop variable.

(point 12 of the coding standard)
https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true